src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
+The src attribute is used in the `<video>` tag to point to the URL of the video to be played. Another way to reference the video is to use the `<source>` element.
+
+One of the key ideas behind the `<source>` element is that the developer can supply multiple video formats to the browser, and the browser will select the first format that the browser understands.
+
+When we look at `<source>` usage, we see that about 40% of videos have no `<source>` element—implying that they use the `src` attribute. This is similar to the ratio found in 2020 (35%).
+
+{{ figure_markup(
+  image="source-element-count.png",
+  caption="`source` element count.",
+  description="A bar chart showing the frequency of various numbers of `source` elements per `video` element. 0.02% of `video` elements contain 5 elements; 0.2% contain 4; 2-3% contain 3; and 7.5% contain 2. The most common number of `source` elements per `video` is 1; around 50% of `video`s contain this many sources. The second most common number of `source` children is 0: 38-40% of `video`s have no `source` children.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=709685412&format=interactive",
+  sheets_gid="1279089769",
+  sql_file="video_number_of_sources_mobile.sql"
+  )
+}}
+
+We also see that the source element is most often used with just one element (50% of all video tags). Only 10% of videos have 2 or more video sources named. By a 3:1 ratio, 2 is more common than 3 sources, and then there is a small selection of more than 3 (there is one video with 48 sources!).
+
+Let's look at the videos that use 2 sources. Here are the top 10 occurrences:
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Format</th>
+      </tr>
+    </thead>
+    </tbody>
+      <tr>
+        <td>`["video/mp4","video/webm"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/webm","video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4","video/ogg"]`</td>
+      </tr>
+      <tr>
+        <td>`[null,null]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4","video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["application/x-mpegURL","video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`[]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4; codecs="avc1.42E01E, mp4a.40.2","video/webm; codecs="vp8, vorbis"`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4;","video/webm;"]`</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="The most common ordered pairs of `type` values, when there are two `source` elements within a `video` element.", sheets_gid="1549760253", sql_file="video_source_formats_mobile.sql") }}</figcaption>
+</figure>
+
+{# TODO I'm not sure this is such an interesting table. Consider adding some stats to make it more interesting? % of usage on mobile and desktop? #}
```

Comment:
@dougsillars should this be a figure table? If so should we add some stats?

Or should it be a bullet list instead?

Reply:
What it looks like here: https://20211207t160821-dot-webalmanac.uk.r.appspot.com/en/2021/media#feature-adoption

Reply:
i think the link is not the right figure, but I found it.  

I can add stats into the table.... like a percentage?

Reply:
Yup. Just seems like a one-column table is more a list than a table. So either change to list or make it at least two columns (or three if wanna show Desktop and mobile). A table does have the advantage that it links to data and SQL query so the latter I think is better.

Reply:
Added the stats in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2705/commits/0b9689ecd8d3a77cfc6de9d8d4de3127933a90c1